owncloudApp/src/main/java/com/owncloud/android/MainApp.java

```diff
@@ -111,23 +112,12 @@ public void onCreate() {
         // initialise thumbnails cache on background thread
         new ThumbnailsCacheManager.InitDiskCacheTask().execute();
 
-        if (BuildConfig.DEBUG || isBeta()) {
-
-            String dataFolder = getDataFolder();
-
-            // Set folder for store logs
-            Log_OC.setLogDataFolder(dataFolder);
-
-            Log_OC.startLogging(Environment.getExternalStorageDirectory().getAbsolutePath());
-            Log_OC.d(BuildConfig.BUILD_TYPE, "start logging " + BuildConfig.VERSION_NAME);
-        }
-
         // register global protection with pass code, pattern lock and fingerprint lock
         registerActivityLifecycleCallbacks(new ActivityLifecycleCallbacks() {
 
             @Override
             public void onActivityCreated(Activity activity, Bundle savedInstanceState) {
-                Log_OC.d(activity.getClass().getSimpleName(), "onCreate(Bundle) starting");
+                Log_OC.v(activity.getClass().getSimpleName(), "onCreate(Bundle) starting");
```

Comment:
Ok, move to more spam. Done

Reply:
`Log_OC.v` would also include this log in the log file so it would be "spam" anyway. What I mean is that .v doesn't fit in this case because Log.v is for Verbose logging and is used when you want to output almost everything. On the other hand, Log.d is used to know the flow of the activities and this is the case.